web-dashboard/public/index.html

```
@@ -0,0 +1,240 @@
+        <h2 id="report-title">Report</h2>
+        <button onclick="closeReport()" class="close-btn">&times;</button>
+    </div>
+    <div id="report-content" class="overlay-body"></div>
+</div>
+
+<div id="toast">Command sent</div>
+
+<script src="app.js"></script>
+</body>
+</html>
+ton>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)